executor/runtime/types/types.go

```diff
@@ -217,32 +241,32 @@ func (c *Container) GetEnv() map[string]string {
 	env["TITUS_NUM_DISK"] = itoa(c.Resources.Disk)
 	env["TITUS_NUM_NETWORK_BANDWIDTH"] = itoa(c.Resources.Network)
 
-	if name := c.TitusInfo.GetImageName(); name != "" {
+	if name := c.imageName; name != "" {
```

Comment:
Maybe add a // TODO: Retrieve these from Docker (Pull)?

Reply:
Sure - not sure if we'd ever actually implement it, though?